images/yugabytedb/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM yugabytedb/yugabyte:latest
```

Comment:
Are you sure you want to use the `latest` image? Better would be the current latest release tag so it stays consistent and reproducible over time.